static/js/validation.js

```diff
@@ -0,0 +1,23 @@
+function validateForm(){
```

Comment:
Please rename to `validateEditItemForm` or similar, since we have multiple forms in this application.